releases/development/master/extra/barclamp_install.rb

```diff
@@ -35,6 +35,17 @@ def usage()
   exit
 end
 
+def naviational_barclamps
+  [
+    "cisco-ucs"
+    "crowbar",
+    "network",
+    "openstack",
+    "swift",
+    "tempest",
```

Comment:
Hrm, I don't think we should hardcode the list here. We should instead check if there's a nav key in the yml file.